docs/workflows.rst

```diff
@@ -477,43 +477,46 @@ Amplitude of low-frequency fluctuation (ALFF) is a measure that ostensibly local
 
-5. The square root of the power spectrum is calculated.
-6. The power spectrum values corresponding to the frequency range retained by the
+4. The square root of the power spectrum is calculated.
+5. The power spectrum values corresponding to the frequency range retained by the
    temporal filtering step are extracted from the full power spectrum.
-7. The mean of the within-band power spectrum is calculated and multiplied by 2.
-8. The ALFF value is multiplied by the standard deviation of the voxel-wise
+6. The mean of the within-band power spectrum is calculated and multiplied by 2.
+7. The ALFF value is multiplied by the standard deviation of the voxel-wise
    ``denoised, interpolated BOLD`` time series.
    This brings ALFF back to its original scale, as if the time series was not normalized.
 
 ALFF will only be calculated if the bandpass filter is enabled
 (i.e., if the ``--disable-bandpass-filter`` flag is not used).
 
-Smoothed ALFF derivatives will also be generated if the ``--smoothing`` flag is used.
+ALFF will also be calculated from the smoothed, denoised BOLD if the ``--smoothing`` flag is used.
 
 
 ReHo
 ----
 :func:`~xcp_d.workflows.restingstate.init_reho_nifti_wf`,
 :func:`~xcp_d.workflows.restingstate.init_reho_cifti_wf`
 
-Regional Homogeneity (ReHo) is a measure of local temporal uniformity in the BOLD signal computed at each voxel of the processed image. 
-Greater ReHo values correspond to greater synchrony among BOLD activity patterns measured in a local neighborhood of voxels, with neighborhood size determined by a user-specified radius of voxels. 
+Regional Homogeneity (ReHo) is a measure of local temporal uniformity in the BOLD signal computed at each voxel of the processed image.
+Greater ReHo values correspond to greater synchrony among BOLD activity patterns measured in a local neighborhood of voxels,
+with neighborhood size determined by a user-specified radius of voxels.
 ReHo is calculated as the coefficient of concordance among all voxels in a sphere centered on the target voxel.
 
-For NIfTIs, ReHo is always calculated via AFNI’s 3dReho with 27 voxels in each neighborhood, using Kendall's coefficient of concordance (KCC). 
-For CIFTIs, the left and right hemisphere are extracted into GIFTI format via Connectome Workbench’s CIFTISeparateMetric. Next, the mesh adjacency matrix is obtained,and Kendall's coefficient of concordance (KCC) is calculated, with each vertex having four neighbors. 
-For subcortical voxels in the CIFTIs, 3dReho is used with the same parameters that are used for NIfTIs. 
+For NIfTIs, ReHo is always calculated via AFNI's 3dReho with 27 voxels in each neighborhood,
+using Kendall's coefficient of concordance (KCC).
+For CIFTIs, the left and right hemisphere are extracted into GIFTI format via
+Connectome Workbench's CIFTISeparateMetric.
+Next, the mesh adjacency matrix is obtained, and Kendall's coefficient of concordance (KCC) is calculated,
+with each vertex having 5-6 neighbors.
```

Comment:
Do we know how similar the size of a 5-6 vertex neighborhood is to a 27 voxel window?

Reply:
27-voxel connectivity would be faces, edges, and corners. I'm having trouble visualizing how the vertices are organized and how they connect.